api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -314,6 +318,24 @@ public Workspace updateWorkspaceAcls(
       }
       throw new BadRequestException(usersNotFound);
     }
+
+    // Finally, keep OWNER and billing project users in lock-step. In Rawls, OWNER does not grant
+    // canCompute on the workspace / billing project, nor does it grant the ability to grant
+    // canCompute to other users.
+    for (String user : Sets.union(updatedAclsMap.keySet(), aclsMap.keySet())) {
+      String fromAccess =
+          aclsMap.getOrDefault(user, new WorkspaceAccessEntry().accessLevel("")).getAccessLevel();
+      WorkspaceAccessLevel toAccess =
+          updatedAclsMap.getOrDefault(user, WorkspaceAccessLevel.NO_ACCESS);
+      if ("OWNER".equals(fromAccess) && WorkspaceAccessLevel.OWNER != toAccess) {
```

Comment:
Will that equivalence work on the right? Shouldn't it be the same format as the left? `!"OWNER".equals(toAccess)`

If so, can we make the string "OWNER" a constant at the top of the file or method?

(Even if we don't need to use owner there, it may make sense to abstract "OWNER" anyway, since we're using it in multiple places)

Reply:
Strings need to use `.equals()`; enums can use `==`. Extracted to an constant though